server/handlers/design_import.go

```diff
@@ -159,7 +160,9 @@ func ConvertFileToDesign(fileToImport FileToImport, registry *registry.RegistryM
 	}
 
 	now = time.Now()
-	design, err := pCore.NewPatternFileFromK8sManifest(manifest, fileToImport.FileName, true, registry)
+	// remove file extension from file name if any
+	name, _ := strings.CutSuffix(sanitizedFile.FileName, sanitizedFile.FileExt)
```

Comment:
this might override the name provided by user ,  we probably dont need this here